### DISCUSSION TITLE
"don" and "kat" under the notes in taiko

### DISCUSSION THREAD

**Discussion by bromomento1348** [ppy/osu#13249] [Ideas]
![image](https://user-images.githubusercontent.com/85076423/120115414-bd0a3d80-c1ad-11eb-9265-6dedc12c55d6.png)
I think we should have this feature because there are some people who want "true" taiko experience like in original game.

**Comment by peppy** [comment]
skin?

**Reply by bromomento1348** [reply]
Doesn't taiko use the same sprite for both don and kat?

**Reply by peppy** [reply]
Maybe? If so this can be proposed as a skin feature for the future.

**Comment by TheFearlessDeath500** [comment]
You mean the kunyomi text that shows below the notes? I think that could be hard-coded in, but idk. peppy’s reply fits more with my response
